pandas/_libs/window.pyx

```diff
@@ -1348,6 +1348,7 @@ def roll_quantile(ndarray[float64_t, cast=True] input, int64_t win,
         bint is_variable
         ndarray[int64_t] start, end
         ndarray[double_t] output
+        double vlow, vhigh
 
     if quantile < 0.0 or quantile > 1.0:
```

Comment:
The code below now relies on 0.0 and 1.0 being filtered by the Python wrapper, doesn't it? We probably should make that explicit here by changing `<` and `>` to `<=` and `>=`.